### PR TITLE
feat: add DetectOrientationScript method

### PR DESCRIPTION
**Pull request by joelrose**
This exposes the DetectOrientationScript method. I've used this for quite some time privately and would like to bring this upstream.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -341,6 +369,24 @@ func (client *Client) HOCRText() (out string, err error) {
...
 }
 
+func (client *Client) DetectOrientationScript() (int, float32, string, float32, error) {
+	if err := client.initOsd(); err != nil {
+		return 0, 0, "", 0, err
+	}
+	var (
+		orient_deg  C.int
+		orient_conf C.float
+		script_name *C.char
```
Comment (otiai10) [inline client.go:379]: 🤔 thinking about memory leak

**Comment by otiai10** [comment]
Alpine test is failing. (please ignore vagrant)

File: client.go (hunk 1 above)
Comment (joelrose) [reply client.go:414]: fixed in: https://github.com/otiai10/gosseract/pull/299/commits/d0f75aa03d096251a845413699d8a6fe82407cc4

**Comment by joelrose** [comment]
Hi @otiai10, thanks for your review! I've fixed the issue, so the alpine test should pass now. Could you review again or merge if all's good? Thanks!

**Comment by otiai10** [comment]
I fixed clearlinux case.

**Review by otiai10 — APPROVED** [review]

**Comment by otiai10** [comment]
Sorry I misunderstood that I fixed your issue by #309

I revert this PR by #310, so please reopen this PR with all test case green.

**Pull request merged**